instaclustr/resource_cluster_unit_test.go

```diff
@@ -499,11 +499,11 @@ func TestHasCassandraSizeChanges(t *testing.T) {
 func TestDoClusterResizeDefault(t *testing.T) {
 	err := doClusterResize(MockApiClient{
 		cluster: Cluster{
-			ID:         "REDIS",
-			BundleType: "REDIS",
+			ID:         "CADENCE",
```

Comment:
Are these changes intentional? What does CADENCE have to do with resizing REDIS?

Reply:
Yes, these changes are intentional. It will test the default behaviour when users want to perform resize of the bundle we haven't supported yet. Previously its Redis, now hand over to CADENCE.